### ISSUE TITLE
Investigate how to make `music` work with WSL

### ISSUE THREAD

**Issue by EliasAAradsson** [lunduniversity/introprog#857]
Project task `music` fails to execute properly on WSL. Consider writing guide on how to fix this problem if possible.

**Comment by bjornregnell** [comment]
At the very least we should warn that it does not work on WSL out of the box.

**Comment by bjornregnell** [comment]
It seems like a mouth-full to get this working:
https://dev.to/pedroafabri/sound-in-wsl2-and-windows-11-pulseaudio-configuration-made-easy-4182
I'm not sure if above instrux are outdated.
Here it says in this old post that MS "has plans" to get it to work out-of-the box: https://www.linuxuprising.com/2021/03/how-to-get-sound-pulseaudio-to-work-on.html


I'm inclined to just state in the compendium that it "may not work in WSL" and leave it up to the student to see if there are workarounds. What do you think @EliasAAradsson ?

**Comment by EliasAAradsson** [comment]
> I'm inclined to just state in the compendium that it "may not work in WSL" and leave it up to the student to see if there are workarounds. What do you think [@EliasAAradsson](https://github.com/EliasAAradsson) ?

Yes, sounds reasonable with just a heads-up about this. After all it's not a course about hacking virtual machines. I'm not that into WSL, so I'm not the best person to ask about this however. It would be interesting to know though if someone gets `music` to work on WSL, so maybe there could be some pointer in the compendium about sharing the knowledge if they get it to work as intended?
